Driverless AI/scoring-pipeline-deployment-to-apache-nifi/scoring-pipeline-deployment-to-apache-nifi.md

```diff
@@ -0,0 +1,352 @@
+
+[4] H2O.ai Community AI Glossary: [Machine Learning Model](https://www.h2o.ai/community/glossary/machine-learning-model)
+
+[5] H2O.ai Community AI Glossary: [Scoring Pipeline](https://www.h2o.ai/community/glossary/scoring-pipeline)
+
+[6] H2O.ai Community AI Glossary: [Model Object, Optimized (MOJO) Scoring Pipeline](https://www.h2o.ai/community/glossary/model-object-optimized-mojo)
+
+[7] [SAVERY - HYDRAULIC TEST RIGS AND BENCHES](https://www.savery.co.uk/systems/test-benches)
+
+[8] [HYDROTECHNIK - Flow and Temperature Testing Components](https://www.hydrotechnik.co.uk/flow-and-temperature-hydraulic-test-bed)
+
+## Prerequisites
+
+- Skilled in Java Object Oriented Programming
+- Driverless AI Environment
+- Driverless AI License
+    - 21 day trial license
+    - It is needed for using the MOJO2 Java Runtime API to execute the MOJO Scoring Pipeline in Apache NiFi
+    - If you need to purchase a Driverless AI license, reach out to our sales team via the [contact us form](https://www.h2o.ai/company/contact/).
+- Basic knowledge of Driverless AI or doing the following tutorials:
+    - [Automatic Machine Learning Introduction with Driverless AI Test Drive Tutorial](https://training.h2o.ai/products/tutorial-1a-automatic-machine-learning-introduction-with-driverless-ai)
+    - [Tutorial 4A: Scoring Pipeline Deployment Introduction](https://training.h2o.ai/products/tutorial-4a-scoring-pipeline-deployment-introduction#tab-product_tab_overview)
+    - [Tutorial 4B: Scoring Pipeline Deployment Templates](https://training.h2o.ai/products/tutorial-4b-scoring-pipeline-deployment-templates)
+    - [Tutorial 4C: Scoring Pipeline Deployment in Java Runtime](https://training.h2o.ai/products/tutorial-4c-scoring-pipeline-execution-in-java-runtime)
+- Downloaded the Driverless AI Deployment Repo to your local machine since we will be using the NiFi Data Flow xml templates that come with mojo-nifi/ folder.
+
+```bash
+git clone -b mojo-nifi https://github.com/james94/dai-deployment-examples/
+```
+
+## Task 1: Set Up Environment
+
+### Create an Environment Directory Structure
+
+```bash
+# Create directory structure for DAI MOJO NiFi Projects
+
+# Create directory where the mojo-pipeline/ folder will be stored
+mkdir $HOME/dai-mojo-nifi/
+
+# Create input directory used for batch scoring or real-time scoring
+mkdir -p $HOME/dai-mojo-nifi/testData/{test-batch-data,test-real-time-data}
+```
+
+### Set Up Driverless AI MOJO Requirements
+
+#### Download MOJO Scoring Pipeline
+
+1\. If you have not downloaded the MOJO Scoring Pipeline, go to [Tutorial 4B: Scoring Pipeline Deployment Templates](https://training.h2o.ai/products/tutorial-4b-scoring-pipeline-deployment-templates), then go to **Task 1: Set Up Environment**, then go to section **Download MOJO Scoring Pipeline** to download it. When finished, come back to this tutorial.
```

Comment:
rewrite this: "then go to **Task 1: Set Up Environment**, then go to section **Download MOJO Scoring Pipeline** to download it."

to this: "then go to **Task 1: Set Up Environment**, then go to the **Download MOJO Scoring Pipeline** section to download it."